examples/cluster-py/__main__.py

```diff
@@ -84,11 +84,15 @@
 ###     EKS Addons     ###
 ##########################
 
+coredns_version = aws.eks.get_addon_version(addon_name="coredns",
+    kubernetes_version=cluster3["eks_cluster"]["version"],
+    most_recent=True)
+
 coredns = eks.Addon(
     f"{project_name}-cluster3-coredns",
     cluster=cluster3,
     addon_name="coredns",
-    addon_version="v1.11.1-eksbuild.9",
+    addon_version=coredns_version.version,
```

Comment:
Unrelated fix looks like.

Reply:
But great! I was fixing this again as port of ops, your solution is more robust.